### ISSUE TITLE
is this thing alive??

### ISSUE THREAD

**Issue by Shemesh** [VersifitTechnologies/angular-ui-tab-scroll#24]
hi,
@seiyria, @smithk58, @tinesoft 

there are some major changes merges and issues that are in waiting and are not handled for some time.
there are some bug fixes...
i (and others like me) are eagerly waiting for some response...

who is the owner of this repository? is it active?

**Comment by seiyria** [comment]
The person you are looking for is @smithk58.

**Comment by Shemesh** [comment]
yes but @smithk58 is not active for a long time now, and this project seems to halt.
@seiyria - is there a chance you can add me as "owner" so i can advance it?

**Comment by seiyria** [comment]
That is not my call. I don't work there anymore.

**Comment by smithk58** [comment]
Hi @Shemesh. Seiyria alerted me to this yesterday and I'm just looking today. This isn't something we're actively maintaining/upgrading, so I've given you write access so you may do so if you wish.

Sorry about the delays!

**Comment by Shemesh** [comment]
@smithk58 tnx for letting the access, you can leave me do the latest changes.

**Comment by Shemesh** [comment]
p.s. i think after the latest changes it is stable and mature enough, so i'll make it version 1.0.0.
sounds good?

**Comment by Shemesh** [comment]
@smithk58

**Comment by Shemesh** [comment]
@smithk58 !!!
i'm so sorry something went terribly wrong with my desktop github, i was in the middle of merging than it crashed and the master branch was deleted. :-(((((
need your help!!
you probably have the master branch on your desktop, can you please bring it back

**Comment by smithk58** [comment]
That seems unlikely. If you want back the version w/ your changes you'll have to do reflog from the looks of it (google "recover deleted git branch").

Otherwise i'll just have to repush whatever version if it I have at home. It wouldn't have the changes you made though - so I would strongly suggest you recover that branch if you want your changes back.

**Comment by Shemesh** [comment]
i can recover my changes (i have them saved aside), but i do not care for them.
for some reason i cannot find the master branch.
is there a way to get it back with all its history?

**Comment by smithk58** [comment]
You can follow the top answer of this: http://stackoverflow.com/questions/5543280/how-do-i-get-the-deleted-branch-back-in-git

TLDR:
git reflog
git checkout -b master ~insert one of the commits from reflog from before you deleted master~

**Comment by Shemesh** [comment]
i'm lost, it says the master branch already exist

**Comment by smithk58** [comment]
You might have created another master on accident. Try:

git checkout -b newMaster ~insert one of the commits from reflog~

Once you have that, just push it and I can put it back on to real master for you.

**Comment by Shemesh** [comment]
pofff... and.... it is back.... got me sweating for a while.

**Comment by smithk58** [comment]
Nice recovery :). I pushed your changes on newMaster to the real master now, as well as deleted newMaster.

**Comment by Shemesh** [comment]
tnx, still no idea how it happened.

**Comment by Shemesh** [comment]
@smithk58 one last thing - in the description on the project (on the very top of the page) there is a link to an old plunk, it seems i don't have permission to change it.
would you please remove the link and leave only plain text, the link to the plunk is in the readme.md

**Comment by smithk58** [comment]
Sure thing. Removed it.

**Comment by Shemesh** [comment]
@smithk58 - please contact me, i need your assistance: shemeshi gmail com
